tools/webdriver/webdriver/bidi/modules/_module.py

```diff
@@ -9,6 +9,8 @@
     TYPE_CHECKING,
 )
 
+from ..undefined import UNDEFINED
```

Comment:
@jrandolf could you maybe make those changes to the bidi client in a separate PR for all the optional values?

Reply:
Sorry, artifacts from a previous PR that _did_ make those changes. I've rebased this PR.